InstabugSample/android/gradle.properties

```diff
@@ -26,3 +26,6 @@ android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
 FLIPPER_VERSION=0.99.0
+
+# Max heap size
+org.gradle.jvmargs=-Xmx4608m
```

Comment:
Should we pass it [from CI environment](https://github.com/Instabug/Instabug-Flutter/blob/be7d9b3b1e55c6a92ae845f28a257b883c33b51d/.circleci/config.yml#L7-L8) instead?